src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -47,14 +49,14 @@ public IEnumerable<string> GetAllFunctionNames()
         }
 
         /// <summary>
-        /// Create a resolver for use in binding. This is called from <see cref="Check(string, FormulaType)"/>.
+        /// Create a resolver for use in binding. This is called from <see cref="Check(string, RecordType)"/>.
         /// Base classes can override this is there are additional symbols not in the config.
         /// </summary>
-        /// <param name="parameterType"></param>
+        /// <param name="alternateConfig">An alternate config that can be provided. Should default to engine's config if null.</param>
```

Comment:
>An alternate config  [](http://example.com/codeflow?start=42&length=20)

What is this for?

Reply:
Used for providing a config that doesn't have display names for globals. We want to be sure we're only changing the name of the field provided, nothing else.